### DISCUSSION TITLE
problem with iptv. unstable connection??

### DISCUSSION THREAD

**Discussion by dlaodmaser** [fabianishere/udm-iptv#265] [Q&A]
hello peeps i have a problem with my KPN iptv and i cant figure it out.  :(

the iptv is connecting and it starts playing tv but after 2 seconds it hangs and gives error STB-NMC-400
(going to another channel gives the same result)

i made a separate vlan just for the iptv box but still the same issue ( i bounded the port to the vlan the id is 2 named "iptv")

this is my config:

## Example configuration for udmp-iptv ##
# Interface on which IPTV traffic enters the router
IPTV_WAN_INTERFACE="eth8"
# ID of VLAN which carries IPTV traffic (use 0 if no VLAN is used)
IPTV_WAN_VLAN="4"
# Name of the IPTV VLAN interface
IPTV_WAN_VLAN_INTERFACE="iptv"
# IP ranges from which the IPTV traffic originates (separated by spaces)
IPTV_WAN_RANGES="213.75.0.0/16  217.166.0.0/16"
# DHCP options to send when requesting an IP address
IPTV_WAN_DHCP_OPTIONS="-O staticroutes -V IPTV_RG"
# LAN interfaces on which IPTV should be made available
IPTV_LAN_INTERFACES=""
# Disable quickleave for igmpproxy
IPTV_IGMPPROXY_DISABLE_QUICKLEAVE="true"
# Enable debugging for igmpproxy
IPTV_IGMPPROXY_DEBUG="false"
IPTV_IGMPPROXY_PROGRAM="igmpproxy"

IPTV_IGMPPROXY_IGMP_VERSION=""

‐----------------------------------------------------

EDIT: 

The diagnose log:

root@UDMPro:~# udm-iptv diagnose
Please share the following output with the developers:
=== Configuration ===
WAN Interface: eth8
WAN VLAN: 4 (dev iptv)
WAN DHCP: true (options "-O staticroutes -V IPTV_RG")
WAN Ranges: 213.75.0.0/16  217.166.0.0/16
LAN Interfaces: 
IGMP Proxy quickleave disabled: true
IGMP Proxy debug: false
=== IP Link and Route ===
28: iptv@eth8:  mtu 1500 qdisc noqueue state UP group default qlen 1000
    inet 10.86.96.140/21 brd 10.86.103.255 scope global iptv
       valid_lft forever preferred_lft forever
10.86.96.0/21 proto kernel scope link src 10.86.96.140 
213.75.112.0/21 via 10.86.96.1 metric 228 
=== Service Logs ===
Jul 07 23:51:42 UDMPro udm-iptvd[1984]: Obtaining IP address for VLAN interface
Jul 07 23:51:42 UDMPro udm-iptvd[2017]: udhcpc: started, v1.30.1
Jul 07 23:51:42 UDMPro udm-iptvd[2017]: udhcpc: sending discover
Jul 07 23:51:43 UDMPro udm-iptvd[2017]: udhcpc: sending select for 10.86.96.140
Jul 07 23:51:43 UDMPro udm-iptvd[2017]: udhcpc: lease of 10.86.96.140 obtained, lease time 3600
Jul 07 23:51:43 UDMPro udm-iptvd[1984]: NATing IPTV network ranges (if necessary)
Jul 07 23:51:43 UDMPro udm-iptvd[1984]: Creating static routes (if necessary)
Jul 07 23:51:43 UDMPro udm-iptvd[1984]: Setting up IGMP Proxy
Jul 07 23:51:43 UDMPro udm-iptvd[1984]: Using igmpproxy...
Jul 07 23:51:43 UDMPro udm-iptvd[1984]: Starting IGMP Proxy

**Comment by wemmenhove** [comment]
I have the same issue...

**Reply by wemmenhove** [reply]
Solved!  My vlan wasn't on the right vlan interface. I see in your diagnostic that there is no vlan interface selected. That could possibly be the same issue. You have to select in the package the right vlan (with the space bar to select/deselect)

**Reply by dlaodmaser** [reply]
Aaaah ye that did it thanks man :P
